examples/tpm2-ekcert/main.go

```diff
@@ -51,6 +47,22 @@ var (
 	}
 )
 
+// Auth Policy calculations in Credential_Profile_EK_V2.0, section 2.1.5.3, "authPolicy"
+func defaultAuthPolicy() []byte {
+	previousDigest := make([]byte, 32)            // All zero buffer
+	policySecretCommandCode := uint32(0x00000151) // TPM__CC_PolicySecret
+	name := uint32(0x4000000B)                    // TPM_RH_ENDORSEMENT
+	policyRef := make([]byte, 0)
+
+	buf := bytes.NewBuffer(previousDigest)
+	binary.Write(buf, binary.BigEndian, policySecretCommandCode)
+	binary.Write(buf, binary.BigEndian, name)
```

Comment:
The above 3 lines are essentially `tpmutil.Pack(make([]byte, 32), policySecretCommandCode, name)`

Reply:
Using `tpmutil.Pack` with a slice automatically prepends the length, so we actually need to Pack the two uint32s then prepend the byte slice.